distsql/distsql_test.go

```diff
@@ -15,31 +15,115 @@ package distsql
 
 import (
 	"github.com/juju/errors"
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/mysql"
+	"github.com/pingcap/tidb/sessionctx/variable"
+	"github.com/pingcap/tidb/store/tikv/oracle"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/charset"
+	"github.com/pingcap/tidb/util/chunk"
 	"github.com/pingcap/tipb/go-tipb"
 	"golang.org/x/net/context"
 )
 
-type mockResponse struct {
-	count int
+func (s *testSuite) TestSelect(c *C) {
+	keyRanges := []kv.KeyRange{
+		{
+			StartKey: kv.Key{0x74, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xf, 0x5f, 0x72, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0},
```

Comment:
Do we need to comment out this key? It's hard to understand.

Reply:
OK